tools/logger/convert.c

```diff
@@ -1052,19 +1052,21 @@ int convert(struct convert_config *config)
 	count = fread(&snd, sizeof(snd), 1, config->ldc_fd);
 	if (!count) {
 		log_err("Error while reading %s.\n", config->ldc_file);
-		return -ferror(config->ldc_fd);
+		ret = -ferror(config->ldc_fd);
```

Comment:
@lyakh I guess one alternative would be to make "struct snd_sof_Logs_header snd" a static global variable. Not much cleaner, but potentially less lines to change.

Reply:
I'd much rather avoid adding more global variables. We already have a couple in the logger and I think we should remove them